platform/security/certificate/src/main/java/org/codice/ddf/certificate/JKSManager.java

```diff
@@ -0,0 +1,207 @@
+    |
+    |  Returns:  None
+    *-------------------------------------------------------------------*/
+
+    public static void addCertToKeyStore(String keyStorePath, String keyStorePassword, String alias,
+            String keyPassword, X509Certificate[] certChain, PrivateKey privKey) {
+        try {
+            KeyStore jks = KeyStore.getInstance("JKS");
+            jks.load(new FileInputStream(keyStorePath), keyStorePassword.toCharArray());
+            jks.setKeyEntry(alias, privKey, keyPassword.toCharArray(), certChain);
+            jks.store(new FileOutputStream(keyStorePath), keyStorePassword.toCharArray());
+        } catch (CertificateException | NoSuchAlgorithmException | IOException | KeyStoreException e) {
+            LOGGER.error(e.getMessage(), e);
+        }
+    }
+
+    /**
+     * ---------------------------------------------------------------------
+     * Method: deleteCertFromKeyStore
+     * <p/>
+     * Purpose:  Deletes the certificate with the given alias from the keystore.
+     * <p/>
+     * Pre-condition:  Keystore contains a certificate with the given alias
+     * <p/>
+     * Post-condition: The keystore file that is output will contain the certificate passed in as a parameter
+     *
+     * @param keyStorePath     path to the keyStore
+     * @param keyStorePassword password for the keystore
+     * @param alias            alias of the certificate to delete
+     *                         -------------------------------------------------------------------
+     */
+
+    public static void removeCertFromKeyStore(String keyStorePath, String keyStorePassword,
+            String alias) {
+        try {
+            KeyStore jks = KeyStore.getInstance("JKS");
+            jks.load(new FileInputStream(keyStorePath), keyStorePassword.toCharArray());
+            jks.deleteEntry(alias);
+            jks.store(new FileOutputStream(keyStorePath), keyStorePassword.toCharArray());
+        } catch (CertificateException | NoSuchAlgorithmException | KeyStoreException | IOException e) {
+            LOGGER.error(e.getMessage(), e);
+        }
+    }
+
+    /**
+     * ---------------------------------------------------------------------
+     * Method: saveX509toNewKeystore
+     * <p/>
+     * Purpose:  Creates a new keystore file and saves an X509 certificate to it.
+     * <p/>
+     * Pre-condition:  None
+     * <p/>
+     * Post-condition: The keystore file that is output will contain the certificate passed in as a parameter
+     *
+     * @param x509       certificate to be saved to the new keystore
+     * @param privateKey the key to be associated with the alias
+     * @param fileName   the name of the file to save
+     * @param alias      alias name
+     * @param password   the password to protect the key
+     *                   -------------------------------------------------------------------
+     */
+
+    public static void saveX509toNewKeystore(X509Certificate x509, PrivateKey privateKey,
```

Comment:
Do we really need this method or should we simply use `createNewKeyStore()` and call `addCert()` on the returned object (assuming we've changed the `createNewKeyStore()` to return an instance of this class)?